src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/PopulatePrisonNumberMigration.kt

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.licences
+
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.prison.PrisonSearchApiClient
+import uk.gov.justice.digital.hmpps.hmppshdcapi.licences.prison.Prisoner
+
+const val UNKNOWN_PRISON_NUMBER = "???"
```

Comment:
idea is to try getting info from prisoner offender search - if we can't find a record in prisoner offender search, we'll mark them with this. 

We could then look at fixing the remaining by direct look up to nomis. This is preferable to battering nomis for all 113,000 licences